src/core/libmaven/groupFiltering.cpp

```diff
@@ -131,4 +131,46 @@ bool GroupFiltering::quantileFilters(PeakGroup *group) {
         return true;
     }
     return false;
+}
+
+void GroupFiltering::deleteDuplicateGroup(){
+    double mzdiff=0.01;
```

Comment:
@mayankmtg all variable names should be in camel case. This is for consistency across this code-base not a general rule.